utils/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -32,11 +32,11 @@ class CopyrightStatementsProcessorTest : WordSpec() {
     init {
         "process" should {
             "return a result with items merged by owner and prefix, sorted by owner and year" {
-                val input = File("src/funTest/assets/copyright-statements.txt").readLines()
+                val input = File("src/test/assets/copyright-statements.txt").readLines()
```

Comment:
Note to myself: So this becomes a "functional" instead of a "unit" test now. For my taste `process()` probably does a little bit too much for a "unit", but I'm fine with this here in order to not introduce a `funTest` source set for `utils`.

Reply:
I don't think this test does too much for a unit test, because it tests the single public function of the class and the class  does not contain dependencies on other code that should be mocked.